### PR TITLE
[v6r17] Fixed some bugs in FreeDiskSpaceCommand

### PR DESCRIPTION
**Pull request by vipersec**

### CODE REVIEW

**Comment by fstagni** [comment]
Please merge as it has conflicts...

File: ResourceStatusSystem/Command/FreeDiskSpaceCommand.py [hunk 1]
```diff
@@ -54,9 +55,11 @@ def doNew( self, masterParams = None ):
...
     se = StorageElement(elementName)
 
-    elementURL = se.getStorageParameters(protocol = "dips")['URLBase']
+    elementURL = se.getStorageParameters(protocol = "dips")
 
-    if not elementURL:
+    if elementURL['OK']:
+      elementURL = se.getStorageParameters(protocol = "dips")['Value']['URLBase']
+    else:
       gLogger.info( "Not a DIPS storage element, skipping..." )
```
Comment (fstagni) [inline ResourceStatusSystem/Command/FreeDiskSpaceCommand.py]: Can you please put this in verbose?

**Comment by fstagni** [comment]
BTW:
```
mysql> SELECT * FROM SpaceTokenOccupancyCache;
+------------------------------------------------------------------+---------------------+---------------+----------------+------------------------+---------------+
| Endpoint                                                         | LastCheckTime       | Guaranteed    | Free           | Token                  | Total         |
+------------------------------------------------------------------+---------------------+---------------+----------------+------------------------+---------------+
| dips://lbcertifdirac6.cern.ch:9196/WorkloadManagement/SandboxSto | 2016-12-06 09:46:15 |             0 |              0 | ProductionSandboxSE    |             0 |
| dips://lhcb-cert-dirac.cern.ch:9196/WorkloadManagement/SandboxSt | 2016-12-06 09:46:15 |             0 |              0 | CertificationSandboxSE |             0 |
```
which means that the "Endpoint" field is too small: it should be set larger.

**Comment by vipersec** [comment]
@fstagni That's right, according to the schema the maximum number of allowed characters is 64.

**Comment by fstagni** [comment]
Yes, modify the schema please (put 128 chars)

**Comment by vipersec** [comment]
@fstagni Ok, I just changed the schema in the certification, should I do the same in production?

**Comment by fstagni** [comment]
No, we'll take care of production. Just mod the cod here now, and add instructions to the wiki.

**Comment by fstagni** [comment]
You still forgot to modify https://github.com/DIRACGrid/DIRAC/blob/rel-v6r17/ResourceStatusSystem/DB/ResourceManagementDB.py#L102 (to 128) and to update the wiki: https://github.com/DIRACGrid/DIRAC/wiki/DIRAC-v6r17

**Pull request merged**